Firmware/RTK_Everywhere/WiFi.ino

```diff
@@ -92,6 +92,8 @@ void menuWiFi()
         systemPrint("m) MDNS: ");
         systemPrintf("%s\r\n", settings.mdnsEnable ? "Enabled" : "Disabled");
 
+        systemPrintf("t) Connect Timeout (ms): %d\r\n", settings.wifiConnectTimeoutMs);
```

Comment:
I don't think requiring the user to think is ms is the correct approach. This will need to be converted to seconds.

Reply:
Also, this setting is very advanced/unlikely to change. It should be placed in a deeper menu. Perhaps System->Software.

Reply:
Done: 8cdcc2be6cee8a8db0d36c90d078c4b316112ff5